packages/testing/src/test/Helpers.test.ts

```diff
@@ -0,0 +1,128 @@
+import { HierarchyCacheMode, initialize, PresentationTestingInitProps, terminate } from "../presentation-testing/Helpers";
+
+describe("Helpers", () => {
+  describe("initialize", () => {
+    let backendInitializationStub: sinon.SinonStub;
+    let frontendInitializationStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      backendInitializationStub = sinon.stub(PresentationBackend, "initialize");
+      frontendInitializationStub = sinon.stub(PresentationFrontend, "initialize");
+      sinon.stub(PresentationBackend, "terminate");
+      sinon.stub(PresentationFrontend, "terminate");
+      sinon.stub(IModelHost, "startup");
+      sinon.stub(NoRenderApp, "startup");
+      sinon.stub(IModelApp, "localization").get(() => ({ getLanguageList: () => ["en"] }));
+    });
+
+    afterEach(async () => {
+      await terminate();
+    });
+
+    it("initializes PresentationBackend and PresentationFrontend on initialize", async () => {
+      await initialize();
+      expect(backendInitializationStub).to.be.calledOnce;
+      expect(frontendInitializationStub).to.be.calledOnce;
+    });
+
+    it("does not initialize PresentationBackend and PresentationFrontend on second call to initialize", async () => {
+      await initialize();
+      expect(backendInitializationStub).to.be.calledOnce;
+      expect(frontendInitializationStub).to.be.calledOnce;
+
+      sinon.resetHistory();
+
+      await initialize();
+      expect(backendInitializationStub).to.not.be.called;
+      expect(frontendInitializationStub).to.not.be.called;
+    });
+
+    it("initializes PresentationBackend and PresentationFrontend with provided props", async () => {
+      const frontendAppStartupSpy = sinon.spy();
+      const props: PresentationTestingInitProps = {
+        backendProps: { id: Guid.createValue() },
+        frontendApp: { startup: frontendAppStartupSpy },
+        frontendProps: { presentation: { clientId: Guid.createValue() } },
+      };
+
+      await initialize(props);
+      expect(backendInitializationStub).to.be.calledOnceWith(props.backendProps);
+      expect(frontendInitializationStub).to.be.calledOnceWith(props.frontendProps);
+      expect(frontendAppStartupSpy).to.be.calledOnce;
+    });
+  });
+
+  describe("terminate", () => {
+    let backendTerminationStub: sinon.SinonStub;
+    let frontendTerminationStub: sinon.SinonStub;
+    let rimrafSyncStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      backendTerminationStub = sinon.stub(PresentationBackend, "terminate");
+      frontendTerminationStub = sinon.stub(PresentationFrontend, "terminate");
+      rimrafSyncStub = sinon.stub(rimraf, "sync");
+      sinon.stub(PresentationBackend, "initialize");
+      sinon.stub(PresentationFrontend, "initialize");
+      sinon.stub(IModelHost, "startup");
+      sinon.stub(NoRenderApp, "startup");
+      sinon.stub(IModelApp, "localization").get(() => ({ getLanguageList: () => ["en"] }));
```

Comment:
it seems both `initialize` and `terminate` suites use the same stubs - maybe move them up a level to avoid repeated code?

Reply:
Moved all except rimrafSyncStub, because it is only used in "terminate" suite.